src/clj/witan/send/params.clj

```diff
@@ -43,9 +43,22 @@
               coll))
           params (sort academic-years)))
 
+(defn modify-transitions-double-joiner-a
+  ; It's unfortunate that the transition format differs between the distribution calculation functions calls.
+  ; Some use raw transitions data, some use transition data having had transition-map called over it.  This means we
+  ; need two versions of this functionality.
+  "Double transitions for setting-A"
+  [transitions]
+  (flatten (for [t transitions]
+             (if (= (:setting-2 t) :A)
+               (repeat 2 t)
+               t))))
+
 (defn calculate-joiners-per-calendar-year
   [transitions]
   (->> (filter transitions-joiner? transitions)
+       ; this modification could be any suitable function
+       (modify-transitions-double-joiner-a)
```

Comment:
Surely this just increases the number of joiners per calendar year. If we want to keep the same amount of students joining SEND each year, but make it "more likely" that they will join a specific setting, we just have to use `modify-transitions-double-joiner-a-transition-map` where the alpha is affected

Reply:
Yep, but this example was to show how to mimic the existing scenario functionality.  And also highlight how we can, if we choose, alter things in a decoupled way i.e only use `modify-transitions-double-joiner-a-transition-map`.

Reply:
ah great! 👍